tests/console/shar.pm

```diff
@@ -19,8 +19,13 @@ use version_utils;
 
 sub run {
     my $self = shift;
-    select_console 'user-console';
 
+    select_console 'root-console';
+    if (is_jeos()) {
+        zypper_call('in sharutils');
```

Comment:
Maybe use a more compact version `zypper_call('in sharutils') if is_jeos();`  or  move `select_console 'root-console';` inside the if.